### PR TITLE
Add Uekibati_mod

### PR DESCRIPTION
**Pull request by Honcharuk**
This mod adds flowerpots for growing plants.

#### Summary

SUMMARY: Content "Adds flowerpots for growing plants"

#### Describe the solution

You can grow crops in pots.

#### Additional context

My first pull request, I will be grateful for the criticism.

### CODE REVIEW

**Comment by mlangsdorf** [comment]
Why is this a mod, instead of part of mainline?
Why are the flowerpots under melee.json?

I'd rather see flowerpots be plantable through a modification of the current iexamine.cpp code, but I understand that if you're not a programmer, that's not a feasible solution.

The JSON needs to be formatted to match the project's standards, please see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md

File: data/mods/Uekibati_mod/recipe_others.json [hunk 1]
```diff
@@ -0,0 +1,1691 @@
...
+  "autolearn": true,
+  "tools": [
+  ],
+  "components": [
+    [
+      [ "fi_ueki_coffee_bean", 1 ]
+    ]
+  ]
+}
+]
```
Comment (ZhilkinSerg) [inline data/mods/Uekibati_mod/recipe_others.json]:
```suggestion
]

```

File: data/mods/Uekibati_mod/melee.json [hunk 2]
```diff
@@ -0,0 +1,1499 @@
...
+    "color": "brown",
+    "price": 5000,
+    "material" : ["clay", "veggy"],
+    "volume": 10,
+    "bashing": 1,
+    "cutting": 0,
+    "to_hit": 0,
+    "category": "tools"
+  }
+]
```
Comment (ZhilkinSerg) [inline data/mods/Uekibati_mod/melee.json]:
```suggestion
]

```

**Comment by ZhilkinSerg** [comment]
Either [lint](http://dev.narc.ro/cataclysm/format.html) mod json files or put them to [json_blacklist](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/json_blacklist).

**Comment by AMurkin** [comment]
You probably should use already existed flower pots with ids `clay_pot_flower` and `plastic_pot_flower` instead of creating a new `uekibati`.

**Pull request closed without merging**